db/migrations/20170825111648_add_name_to_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = (Knex, Promise) => {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title SET NOT NULL');
```

Comment:
isn't the `title`column already `not null`?

Reply:
oops meant to drop the `not null` constraint